code/game/objects/structures/icemoon/cave_entrance.dm

```diff
@@ -169,7 +169,30 @@ GLOBAL_LIST_INIT(ore_probability, list(
 		if(17)
 			new /obj/item/reagent_containers/food/drinks/drinkingglass/filled/nuka_cola(loc)
 		if(18)
-			new /obj/item/assembly/signaler/anomaly/bluespace(loc)
+			//Here we spawn 23 empty bags of cheese honkers
+			new /obj/item/trash/cheesie(loc)
+			new /obj/item/trash/cheesie(loc)
+			new /obj/item/trash/cheesie(loc)
+			new /obj/item/trash/cheesie(loc)
+			new /obj/item/trash/cheesie(loc)
+			new /obj/item/trash/cheesie(loc)
+			new /obj/item/trash/cheesie(loc)
+			new /obj/item/trash/cheesie(loc)
+			new /obj/item/trash/cheesie(loc)
+			new /obj/item/trash/cheesie(loc)
+			new /obj/item/trash/cheesie(loc)
+			new /obj/item/trash/cheesie(loc)
+			new /obj/item/trash/cheesie(loc)
+			new /obj/item/trash/cheesie(loc)
+			new /obj/item/trash/cheesie(loc)
+			new /obj/item/trash/cheesie(loc)
+			new /obj/item/trash/cheesie(loc)
+			new /obj/item/trash/cheesie(loc)
+			new /obj/item/trash/cheesie(loc)
+			new /obj/item/trash/cheesie(loc)
+			new /obj/item/trash/cheesie(loc)
+			new /obj/item/trash/cheesie(loc)
+			new /obj/item/trash/cheesie(loc)
```

Comment:
```suggestion
			new /obj/item/raw_anomaly_core/bluespace(loc)
```

Reply:
I'd suggest adding a check to see if the anomaly core cap has been reached and dropping an activated core if it hasn't, but I'm a bit worried that a miner (or even just a portal or demon somehow dying on their own) could "soak up" parts of the bluespace anomaly core limit accidentally.